AlphaWallet/Accounts/Coordinators/AccountsCoordinator.swift

```diff
@@ -182,18 +181,23 @@ class AccountsCoordinator: Coordinator {
             }
             controller.addAction(renameAction)
 
-            let copyAction = UIAlertAction(title: R.string.localizable.copyAddress(), style: .default) { [navigationController] _ in
-                UIPasteboard.general.string = account.address.eip55String
-                navigationController.view.showCopiedToClipboard(title: R.string.localizable.copiedToClipboard())
+            let copyAction = UIAlertAction(title: R.string.localizable.copyAddress(), style: .default) { [weak self] _ in
+                self?.showCopiedToClipboard(account: account)
             }
             controller.addAction(copyAction)
+
             let cancelAction = UIAlertAction(title: R.string.localizable.cancel(), style: .cancel) { _ in }
             controller.addAction(cancelAction)
 
             navigationController.present(controller, animated: true)
         }
     }
 
+    private func showCopiedToClipboard(account: Wallet) {
```

Comment:
Maybe `copyToClipboard(account:)` instead? Because it doesn't only show the alert, it does the copying too